decidim-core/app/packs/src/decidim/redesigned_index.js

```diff
@@ -112,47 +111,25 @@ Rails.start()
  * @returns {void}
  */
 const initializer = (element = document) => {
-  window.theDataPicker = new DataPicker($(".data-picker"));
+  window.theDataPicker = new DataPicker($(".data-picker", element));
 
   let focusContainer = element;
   if (element === document) {
     focusContainer = document.querySelector("body");
   }
   window.focusGuard = new FocusGuard(focusContainer);
 
+  // REDESIGN_PENDING: deprecated
   $(element).foundation();
-  $(element).on("open.zf.reveal", (ev) => {
```

Comment:
kill foundation 🔪

Reply:
Not yet, that will tear the admin site apart.